app/javascript/dashboard/routes/dashboard/settings/reports/components/Filters/v3/FilterDropdownSearch.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
+defineProps({
+  buttonText: {
+    type: String,
+    default: '',
+  },
+  inputValue: {
+    type: String,
+    default: '',
+  },
+});
+</script>
+<template>
+  <div
+    class="flex items-center justify-between h-10 min-h-[40px] sticky top-0 bg-white z-10 dark:bg-slate-800 gap-2 px-3 border-b rounded-t-xl border-slate-50 dark:border-slate-700"
+  >
+    <div class="flex items-center gap-2">
+      <fluent-icon
+        icon="search"
+        size="18"
+        class="text-slate-400 dark:text-slate-400"
+      />
+      <input
+        type="text"
+        class="w-full mb-0 text-sm bg-white dark:bg-slate-800 text-slate-800 dark:text-slate-75 reset-base"
+        placeholder="Search"
```

Comment:
Need to use translated text

Reply:
@scmmishra I will move it to props. Thoughts? So we can reuse.